proto/hub-genesis/tx.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+package rollapp.hub_genesis;
+
+import "cosmos/msg/v1/msg.proto";
+import "google/protobuf/any.proto";
+
+option go_package = "github.com/dymensionxyz/dymension-rdk/x/hub-genesis/types";
+
+service Msg {
+  rpc SendTransfer(MsgSendTransfer) returns (MsgSendTransferResponse);
+}
+
+// Try to send the genesis transfer
+// Must be whitelisted relayer, and channel must be open
+// Must have tokens available to send
+message MsgSendTransfer {
+  option (cosmos.msg.v1.signer) = "signer";
+  string signer = 1;
+  // ID of the canonical channel, as queried from the hub
+  string channel_id = 2;
```

Comment:
I have strong believe we should always have port-id and channel-id together

Reply:
https://github.com/dymensionxyz/dymension-rdk/issues/629

Reply:
After think three, I think this is good, as we always force it to use the `transfer`-IBC thing instead of handling dynamic cases like `ica` or whatever. Wdyt?

Reply:
I'm happy with that